commit 27cdf54f89952229ba533f4c7184c4e29c14e4c6
Author: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>
Date:   Tue Aug 21 21:40:16 2018 +0200

    ibutils fix build dependency
    
    Building with -j1 fails as ibis is built before libibiscom.a
    
    Signed-off-by: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>

diff --git ibis/src/Makefile.am ibis/src/Makefile.am
index 9e1bfa21a278..87064954e4ea 100644
--- ibis/src/Makefile.am
+++ ibis/src/Makefile.am
@@ -76,6 +76,7 @@ ibis_SOURCES = ibissh_wrap.cpp
 
 # note the order of the libraries does matter as we static link
 ibis_LDADD = -L. -libiscom $(OSM_LDFLAGS) $(TCL_LIBS)
+ibis_DEPENDENCIES = libibiscom.a
 
 
 # SWIG FILES:
